lib/bemxjst/match.js

```diff
@@ -48,7 +48,7 @@ function MatchCustom(template, pred) {
 }
 
 MatchCustom.prototype.exec = function exec(context) {
-  return this.body.call(context);
+  return this.body.call(context, context);
```

Comment:
Chrome 49 and  already has that. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Browser_compatibility

Let's just wait a little ;-)